oak_functions/client/rust/src/rekor.rs

```diff
@@ -20,19 +20,23 @@
 use anyhow::Context;
 use ecdsa::Signature;
 use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha256};
 use signature::Verifier;
-use std::str::FromStr;
+use std::{cmp::Ordering, str::FromStr};
+
+/// Number of bytes required for storing a SHA256 hash.
+pub const SHA256_HASH_LENGTH: usize = 32;
```

Comment:
nit: we only use it once. Do we need a `pub` constant? I think we could also just inline and document where used.